File: SingleTapJavaSample/app/src/main/java/com/example/singletapjavasample/SubActivity.java
package com.example.singletapjavasample;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.Toast;

public class SubActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_sub);
        Toast.makeText(this, "SubActivity Create!", Toast.LENGTH_SHORT).show();
    }
}

File: SingleTapKotlinSample/app/src/main/java/com/leo/singletapsample/SubActivity.kt
package com.leo.singletapsample

import android.os.Bundle
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity

class SubActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        Toast.makeText(this, "SubActivity onCreate", Toast.LENGTH_SHORT).show()
    }
}
